SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -167,6 +168,9 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	bool bWaitingForAcceptingPlayersToSpawn;
 	FString SnapshotToLoad;
 
+	uint32 ActorReplicationRateLimit = GetDefault<USpatialGDKSettings>()->ActorReplicationRateLimit;
```

Comment:
These don't need to be stored here. Let's just get them from the settings in `ServerReplicateActors_ProcessPrioritizedActors`.